aten/src/ATen/cuda/tunable/GemmCommon.h

```diff
@@ -98,15 +122,30 @@ struct GemmParams : OpParams {
   at::opmath_type<T> beta;
   T* c;
   int64_t ldc;
+private:
+  bool deep;
```

Comment:
it seems "deep" has the same meaning as "duplicate_inputs" parameter in the other functions when a and b were allocated bytes, and then need to be deleted. Maybe use the same name?

Reply:
Done as suggested.